### DISCUSSION TITLE
Compatibility with GitOps & ArgoCD

### DISCUSSION THREAD

**Discussion by rowi1de** [operator-framework/operator-lifecycle-manager#3049] [Q&A]
After digging around how to install OLM via ArgoCD I came along this thread: https://github.com/operator-framework/operator-lifecycle-manager/issues/829#issuecomment-939416777

After that, I was wondering how GitOps friendly operator-lifecycle-manager is in general? Is someone using it with ArgoCD?

**Comment by TigranKhudav** [comment]
@dinhxuanvu , @ecordell Any news about OLM official Helm chart or how to install OLM using ArgoCD ?

**Reply by dinhxuanvu** [reply]
As far as I know, operator-lifecycle-manager (OLM v0) is not designed to work with an external CI/CD extension like ArgoCD. Given OLM v0 is no longer in active development, I doubt there is gonna be a support for ArgoCD in the future. You can check out the new OLM v1 (https://github.com/operator-framework/operator-controller) which I don't think work with ArgoCD either at the moment but you can ask for it as a feature given it is an active project.
